pom.xml

```diff
@@ -177,7 +177,7 @@
     <audienceannotations.version>0.15.1</audienceannotations.version>
     <clp-ffi.version>0.4.7</clp-ffi.version>
     <stax2-api.version>4.2.2</stax2-api.version>
-    <aws.sdk.version>2.31.48</aws.sdk.version>
+    <aws.sdk.version>2.31.46</aws.sdk.version>
```

Comment:
Please revert the version changes. They are unrelated

Reply:
I don't know which automatism modified these versions, but I'm sure it wasn't my intention.